CONTRIBUTING.md

```diff
@@ -41,3 +41,9 @@ make check_lint
 ```shell
 pre-commit install
 ```
+
+## Build the docs locally
+
+```shell
+make html
+```
```

Comment:
```suggestion
```

To delete all intermediate files and cached content and build the docs from scratch, run `make cleandocs` before `make html`
```